numpy/ma/extras.py

```diff
@@ -257,10 +252,10 @@ def getdoc(self):
         npfunc = getattr(np, self.__name__, None)
         doc = getattr(npfunc, '__doc__', None)
         if doc:
-            sig = self.__name__ + ma.get_object_signature(npfunc)
-            locdoc = "Notes\n-----\nThe function is applied to both the _data"\
-                     " and the _mask, if any."
-            return '\n'.join((sig, doc, locdoc))
+            sig = self.__name__ + ma.get_object_signature(npfunc) + '\n'
+            doc = ma.doc_note(doc, "The function is applied to both the _data"
+                                   " and the _mask, if any.")
+            return '\n'.join((sig, doc))
```

Comment:
No point adding `\n` in two different places:
```suggestion
            sig = self.__name__ + ma.get_object_signature(npfunc)
            doc = ma.doc_note(doc, "The function is applied to both the _data "
                                   "and the _mask, if any.")
            return '\n\n'.join((sig, doc))
```